vip-api/src/main/java/fr/insalyon/creatis/vip/api/controller/DTO/SignUpUserDTO.java

```diff
@@ -0,0 +1,116 @@
+package fr.insalyon.creatis.vip.api.controller.DTO;
```

Comment:
Please put this class in the *.vip.api.model package

Reply:
fixed